core/common/src/main/java/alluxio/RuntimeConstants.java

```diff
@@ -33,8 +33,11 @@
   public static final String ALLUXIO_JAR =
       "target/alluxio-" + VERSION + "-jar-with-dependencies.jar";
 
-  /** The URL to Alluxio documentation for this version on project web site. */
+  /** The URL of Alluxio documentation for this version on project web site. */
   public static final String ALLUXIO_DOCS_URL;
 
+  /** The URL of Alluxio debugging documentation. */
+  public static final String ALLUXIO_DEBUG_DOCS_URL = ALLUXIO_DOCS_URL + "/en/Debugging-Guide.html";
```

Comment:
according to [Java Spec](http://docs.oracle.com/javase/specs/jls/se8/html/jls-12.html#jls-12.4.2), the initializers (static block or field) are executed by text order in source code